library/core/src/intrinsics/mir.rs

```diff
@@ -249,6 +249,11 @@
 //!  - [`Call`] has an associated function as well. The third argument of this function is a normal
 //!    function call expression, for example `my_other_function(a, 5)`.
 //!
+//! #### Debuginfo
+//!
+//!  - A debuginfo name can be given to a local using `debug my_name => contents;`.
+//!    For `contents`, we use the same syntax as operands, to support both places and constants.
```

Comment:
I have no idea what this means. How do I use an operand to set debuginfo...?

An example would help.